src/main/java/codeu/controller/LoginServlet.java

```diff
@@ -84,6 +84,10 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
     }
 
     request.getSession().setAttribute("user", username);
+    if (username.equals("anAdmin")){
+        response.sendRedirect("/admin");
+        return;
+    }
```

Comment:
I think we may want to remove this if statement because admins should still be able to arrive at the conversations page after logging in. With the if statement they would always arrive at the admin page instead.